packages/turbo-ignore/src/index.ts

```diff
@@ -99,21 +100,3 @@ exec(
     }
   }
 );
-
-export function getComparison(): {
```

Comment:
Moved this function out for testing - this file is executable so you can't import from it to test each piece independently without the entire file executing.